hw02/test result hw02.txt

```diff
@@ -0,0 +1,60 @@
+{\rtf1\ansi\ansicpg936\cocoartf2709
+\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fnil\fcharset0 Menlo-Regular;}
+{\colortbl;\red255\green255\blue255;\red0\green0\blue0;}
+{\*\expandedcolortbl;;\csgray\c0;}
+\paperw11900\paperh16840\margl1440\margr1440\vieww11520\viewh8400\viewkind0
+\pard\tx560\tx1120\tx1680\tx2240\tx2800\tx3360\tx3920\tx4480\tx5040\tx5600\tx6160\tx6720\pardirnatural\partightenfactor0
+
+\f0\fs22 \cf2 \CocoaLigature0 wyakengp@yakengpdeMacBook-Air cs61a % python3 test.py hw/HW02/hw02.py product 2 3 identity 5 identity 3 square 5 square 3 increment 3 triple\
+INFO:root:Calling function 'product' with arguments [3, <function <lambda> at 0x1049db160>] (call 1)\
```

Comment:
Above this line, remove those text which are not related to test result.

Ideally, the test result should show only text:

INFO:root:Calling function 'double_eights' with arguments [80808080] (call 1)
True

INFO:root:Calling function 'double_eights' with arguments [8080] (call 1)
True

This rule applies to other places in the test result file.